web/modules/custom/nys_bill_vote/nys_bill_vote.module

```diff
@@ -49,13 +49,21 @@ function nys_bill_vote_node_view_alter(array &$build, EntityInterface $entity, E
       'entity_type' => $entity->bundle(),
       'entity_id' => $entity->id(),
       'is_embed' => empty($route_node) || $route_node->id() !== $entity->id(),
+      'simple_mode' => FALSE,
     ];
 
     if ($display->getComponent('vote_widget')) {
       $build['vote_widget'] = [
         'form' => \Drupal::formBuilder()->getForm('Drupal\nys_bill_vote\Form\BillVoteWidgetForm', $bill_vote_form_settings),
       ];
     }
+
+    if ($display->getComponent('vote_widget_simple')) {
```

Comment:
Here are the two options I considered, and why I passed on them:

1. Hardwire the `simple_mode` config to the `my_dashboard_teaser` display mode through a form alter -- I thought it might be useful to have this available in the UI, without more form alters, but I can be talked into this.
2. Create a full field plugin for the bill vote field, with `simple_mode ` as a config option (as mentioned, the existing pseudo-field approach doesn't allow for UI configuration -- this just felt like overkill for a field that doesn't otherwise store data

Happy to reconsider any of these (with #1 being my preference, if we want to do away with the additional "simple" pseudo-field.